package.json

```
@@ -12,13 +12,17 @@
   "module": "dist/index.js",
   "scripts": {
     "start": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds\"",
+    "prepare:ghpages": "rimraf ghpages && yarn run prepare && vite build",
```

there is: `prepare:ghpages` and `ghpages:prepare` - let's keep the `ghpages:` prefix and update this one to add to group below. `ghpages:build`
```suggestion
    "ghpages:build": "rimraf ghpages && yarn run prepare && vite build",
```